scripts/kafka_bridge_config_generator.sh

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env bash
+
+# Write the config file
+
+SECURITY_PROTOCOL=PLAINTEXT
+
+if [ "$KAFKA_BRIDGE_TLS" = "true" ]; then
+    SECURITY_PROTOCOL="SSL"
+
+    if [ -n "$KAFKA_BRIDGE_TRUSTED_CERTS" ]; then
+        TLS_CONFIGURATION=$(cat <<EOF
+# TLS / SSL
+kafka.ssl.truststore.location=/tmp/kafka/bridge.truststore.p12
+kafka.ssl.truststore.password=${CERTS_STORE_PASSWORD}
+kafka.ssl.truststore.type=PKCS12
+EOF
+)
+    fi
+
+    if [ -n "$KAFKA_BRIDGE_TLS_AUTH_CERT" ] && [ -n "$KAFKA_BRIDGE_TLS_AUTH_KEY" ]; then
+        TLS_AUTH_CONFIGURATION=$(cat <<EOF
+kafka.ssl.keystore.location=/tmp/kafka/bridge.keystore.p12
+kafka.ssl.keystore.password=${CERTS_STORE_PASSWORD}
+kafka.ssl.keystore.type=PKCS12
+EOF
+)
+    fi
+fi
+
+if [ -n "$KAFKA_BRIDGE_SASL_USERNAME" ] && [ -n "$KAFKA_BRIDGE_SASL_PASSWORD_FILE" ]; then
+    if [ "$SECURITY_PROTOCOL" = "SSL" ]; then
+        SECURITY_PROTOCOL="SASL_SSL"
+    else
+        SECURITY_PROTOCOL="SASL_PLAINTEXT"
+    fi
+
+    PASSWORD=$(cat /opt/bridge/bridge-password/$KAFKA_BRIDGE_SASL_PASSWORD_FILE)
+
+    if [ "x$KAFKA_BRIDGE_SASL_MECHANISM" = "xplain" ]; then
+        SASL_MECHANISM="PLAIN"
+        JAAS_SECURITY_MODULE="plain.PlainLoginModule"
+    elif [ "x$KAFKA_BRIDGE_SASL_MECHANISM" = "xscram-sha-512" ]; then
+        SASL_MECHANISM="SCRAM-SHA-512"
+        JAAS_SECURITY_MODULE="scram.ScramLoginModule"
+    fi
+
+    SASL_AUTH_CONFIGURATION=$(cat <<EOF
+kafka.sasl.mechanism=${SASL_MECHANISM}
+kafka.sasl.jaas.config=org.apache.kafka.common.security.${JAAS_SECURITY_MODULE} required username="${KAFKA_BRIDGE_SASL_USERNAME}" password="${PASSWORD}";
+EOF
+)
+fi
+
+content="#Common properties\n"
```

Comment:
This is super ugly, you should do it do the same as you are doing above I think.